Octokit.Reactive/Clients/ObservableGitHubAppsClient.cs

```diff
@@ -175,5 +175,19 @@ public IObservable<Installation> GetUserInstallationForCurrent(string user)
 
             return _client.GetUserInstallationForCurrent(user).ToObservable();
         }
+
+        /// <summary>
+        /// Creates a GitHub app by completing the handshake necessary when implementing the GitHub App Manifest flow.
+        /// https://docs.github.com/apps/sharing-github-apps/registering-a-github-app-from-a-manifest
+        /// </summary>
+        /// <remarks>https://docs.github.com/rest/apps/apps#create-a-github-app-from-a-manifest</remarks>
+        /// <param name="code">Temporary code in a code parameter.</param>
+        [ManualRoute("POST", "/app-manifests/{code}/conversions")]
```

Comment:
Do you mind explaining why the `ManualRoute` decorator is required here?

Reply:
leftover copy/pasta.  removed